src/map/packet_system.cpp

```diff
@@ -315,12 +315,15 @@ void SmallPacket0x00A(map_session_data_t* const PSession, CCharEntity* const PCh
         }
         PChar->status = STATUS_TYPE::NORMAL;
     }
-    else
+    else if (PChar->loc.zone != nullptr)
     {
-        if (PChar->loc.zone != nullptr)
-        {
-            ShowWarning("Client cannot receive packet or key is invalid: %s", PChar->GetName());
-        }
+        ShowWarning("Client cannot receive packet or key is invalid: %s, Zone: %s (%i)",
+            PChar->GetName(), PChar->loc.zone->GetName(), PChar->loc.zone->GetID());
+
+        // Write a sane location for them
+        auto newZone = PChar->loc.prevzone ? PChar->loc.prevzone : ZONE_VALKURM_DUNES;
```

Comment:
So, this doesn't nicely forward someone out of death into a real zone, but when they eventually get bored of their black loading screen and restart the game, they'll appear in a sane zone

Reply:
We normally used 214 for that, which is basically the jeuno mog house. we have some magic in core attached to it that will dump you to a valid prevZone on exit because we direct you landed there "abnormally".

its kinda obscure you'd prolly never notice it without me telling the story. :)